### DISCUSSION TITLE
請問能夠單獨執行虛構敘事嗎?

### DISCUSSION THREAD

**Discussion by akuan1997** [moesnow/March7thAssistant#143] [General]
逐光撿金都會從忘卻之廷開始 
不論我設定檔案裡面有沒有開啟
但是我目前還沒有辦法滿星 這樣就會一直卡在忘卻無法執行虛構敘事
我想請問有辦法能夠直接運行虛構敘事嗎?
還是一定要透過完整運行或逐光撿金呢?

**Comment by moesnow** [comment]
以后会添加单独启动的选项

**Reply by akuan1997** [reply]
好的 謝謝你!
